modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -0,0 +1,42 @@
+[[quickstart-sap-setup]]
+= Setting Up
+
+Your SAP cluster requires several client systems.
+Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready on your {productname} Server.
+You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+
+One of the key features of SAP is high availability of the cluster.
+Every component within an SAP cluster has redundancy and failover protection.
+When you are preparing your clients, ensure you have enough hardware and infrastructure to allow for this.
```

Comment:
I think we document this minimal requirement somewhere, but I don't honestly know

Reply:
Found it in the SAP Guide. Added.